pootle/apps/pootle_project/migrations/0011_add_none_checkstyle.py

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.9.9 on 2016-09-22 05:17
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('pootle_project', '0010_add_reserved_code_validator'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='project',
+            name='checkstyle',
+            field=models.CharField(choices=[(b'creativecommons', b'creativecommons'), (b'drupal', b'drupal'), (b'gnome', b'gnome'), (b'kde', b'kde'), (b'libreoffice', b'libreoffice'), (b'mozilla', b'mozilla'), (b'none', b'none'), (b'openoffice', b'openoffice'), (b'standard', b'standard'), (b'terminology', b'terminology'), (b'wx', b'wx')], default=b'standard', max_length=50, verbose_name='Quality Checks'),
```

Comment:
Somehow I thought we'd undone this brokenness.  This would mean needing to change the model every time you add a checker which seems like the wrong approach.  I'm not sure how the customer checker is impacted either.